docs/source/trainer/schedulers.rst

```diff
@@ -76,12 +74,60 @@ Below are the supported schedulers found at :mod:`composer.optim.scheduler`.
     CosineAnnealingWarmRestartsScheduler
     PolynomialScheduler
 
+.. note::
+
+    Compared to pytorch schedulers, :class:`.ComposerScheduler` need not be provided
+    an ``optimizer`` directly. The trainer will handle binding the optimizer when
+    converting our schedulers to :class:`~torch.optim.lr_scheduler.LabmdaLR`.
```

Comment:
I think this is a little more detail than needs to be given. We can just say something like "The training will attach the optimizer later when it compiles the scheduler."